.github/workflows/integration-tests.yml

```diff
@@ -574,7 +576,7 @@ jobs:
   ### Start Live Testnet Section
 
   testnet-smoke-tests-matrix:
-    if: startsWith(github.ref, 'refs/tags/') ## Only run live tests on new tags
+    if: startsWith(github.ref, 'refs/tags/') || ${{ github.event_name == 'schedule' }} ## Only run live tests on new tags and nightly
```

Comment:
Would it make sense to add a step which will check for funds in the wallet before triggering tests? will make debugging easier.

Reply:
That makes sense. Will make a PR to our GHA repo.